### PR TITLE
Role files

### PR DESCRIPTION
**Pull request by jbzdak**
This is a part of `plebia/common` role related to apt configuration.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - apticron
+  - apt-file
+  - etckeeper
+  - links
+  - tree
+  - pwgen
+  - libxml2-dev
+  - libxslt1-dev
+  - python-dev
+  - tmpreaper
```
Comment (antoviaque) [inline defaults/main.yml]: For etckeeper and tmpreaper, just installing it isn't enough - they also needs to be configured. I saw that you included the configuration in another role - which is fine, but then the package dependency must be declared by the role configuring it.

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- name: Autoremove unused packages
+  command: apt-get -y autoremove
+  when: "'packages will be REMOVED' in check_autoremove.stdout"
+
+- name: Install prerequisite packages
+  apt: pkg={{ item }} state=installed update_cache=yes cache_valid_time=18000
+  with_items: "{{ ROLE_INDEPENENT_USEFULL_PACKAGES }}"
+
+- name: configure unattened upgrades
+  template: src=unattended-upgrades dest=/etc/apt/apt.conf.d/50unattended-upgrades
```
Comment (antoviaque) [inline tasks/main.yml]: This is handled by `geerlingguy.security`: https://github.com/antoviaque/plebia-configuration/blob/7b19b5bf0cf4d8dc2478984653a3b2801542ee2c/roles/geerlingguy.security/tasks/autoupdate-Debian.yml

Comment (jbzdak) [inline tasks/main.yml]: Unfortunatelly not :( You wanted to have emails in case of errors so I added it here.

Comment (antoviaque) [inline tasks/main.yml]: What isn't handled by `geerlingguy.security` exactly? Sending the emails in case of failure?

Comment (jbzdak) [inline tasks/main.yml]: [Contents of this file](https://github.com/geerlingguy/ansible-role-security/blob/master/templates/50unattended-upgrades.j2) in the security role is as follows: 

```
Unattended-Upgrade::Automatic-Reboot "false";

Unattended-Upgrade::Allowed-Origins {
        "${distro_id} ${distro_codename}-security";
//      "${distro_id} ${distro_codename}-updates";
};
```

You need to add following lines to get an e-mail on failure: 

```
Unattended-Upgrade::Mail "root";
Unattended-Upgrade::MailOnlyOnError "true"; 
```

Comment (antoviaque) [inline tasks/main.yml]: @jbzdak Ok - so can we contribute that change to `geerlingguy.security`, maybe activated with an option if needed to get it merged there? That would avoid having to maintain our own version of this, and could be helpful to others.

Comment (jbzdak) [inline tasks/main.yml]: @antoviaque I can do a PR against `security repository` :)

Comment (antoviaque) [inline tasks/main.yml]: Great : )

**Comment by antoviaque** [comment]
:+1:

**Pull request merged**